src/indicator/algorithm/calculator/stochasticOscillator.js

```diff
@@ -56,13 +56,8 @@ export default function() {
         return stochastic;
     };
 
-    rebind(stochastic, kWindow, {
-        kWindowSize: 'windowSize'
-    });
-
-    rebind(stochastic, dWindow, {
-        dWindowSize: 'windowSize'
-    });
+    rebindAll(stochastic, kWindow, map({'windowSize': 'kWindowSize'}));
+    rebindAll(stochastic, dWindow, map({'windowSize': 'dWindowSize'}));
```

Comment:
Actually, I presume that a `map` is an implicit `include`? That's not immediately obvious.